images/builder/docker/custom-docker-builder/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 # The standard name for this image is openshift/origin-custom-docker-builder
 #
-FROM openshift/origin-base
+FROM openshift/origin-base:v1.5
```

Comment:
@stevekuznetsov @tdawson 
We are updating the build process to accommodate the tagging issue in FROM stanza[1].
Once this is in, i will update this PR to remove the tags from Dockerfiles

[1] https://github.com/CentOS/container-pipeline-service/pull/330